gsoc2018-bharat/src/evaluate.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+from numpy.linalg import norm
+import json
+from gensim.models import FastText
+# from gensim.models import KeyedVectors
+import logging
+import sys
+import torch
+import Encoder
+from torch.autograd import Variable
+from sklearn.metrics.pairwise import cosine_similarity
+
+
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
+punctuation = '!"#$%&\'()*+,.:;<=>?@[\\]^`{|}~'
+table = str.maketrans('', '', punctuation)
+dictionary = {}
+model = FastText.load('model/entity_fasttext_n100')
+wv = model.wv
+del model
+# f = '/Volumes/Seagate/SeagateBackupPlus/Github/GSoC2018/glove.6B/glove.6B.100d.txt'
```

Comment:
Ok, so you need to thoroughly review this before you PR. General issues:

1. Lots of commented out code 
2. Lots of `global` uses of variables within a function when the function docstring mentions that the variable is passed as an arg . You need to decide which is the case, passing data as args is usually much better than declaring global states